examples/plot_model_card.py

```diff
@@ -60,49 +61,64 @@
 # Create a model card
 # ====================
 # We now create a model card, set couple of attributes and save it.
-# We first set the metadata with CardData and pass it to create_model_card.
-# Then, we pass information other than metadata in kwargs.
+# Then, we pass information using add() and plots using add_inspection()
 # We'll initialize a local repository and save the card with the model in it.
 
+model_card = card.Card(model)
+
+
+# %%
+# Initialize a repository to save our files in
+# ============================================
+# We will now initialize a repository and save our model
```

Comment:
We should first initialize the local repo, then create the modelcard, which also would be inline with https://github.com/skops-dev/skops/pull/45